### PR TITLE
LightShadow: Don't export classes.

### LINKED ISSUE #20109: PointLightShadow is not exported
##### Description of the problem

https://github.com/mrdoob/three.js/blob/5a8c62fe37420429377de25885acf8878f2bcb1e/src/Three.js#L56-L63 LightShadow, DirectionalLightShadow and SpotLightShadow are exported but not **PointLightShadow**.

##### Three.js version

- [X] Dev

##### Browser

- [x] All of them
##### OS

- [x] All of them

##### Hardware Requirements (graphics card, VR Device, ...)

N/A

### PR DESCRIPTION
**Pull request by Mugen87**
Fixed #20109.

### CODE REVIEW

File: src/Three.d.ts [hunk 1]
```diff
@@ -50,15 +50,12 @@ export * from './loaders/Loader';
...
 export * from './loaders/AudioLoader';
-export * from './lights/SpotLightShadow';
 export * from './lights/SpotLight';
 export * from './lights/PointLight';
 export * from './lights/RectAreaLight';
 export * from './lights/HemisphereLight';
-export * from './lights/DirectionalLightShadow';
 export * from './lights/DirectionalLight';
 export * from './lights/AmbientLight';
-export * from './lights/LightShadow';
```
Comment (donmccurdy) [inline src/Three.d.ts:61]: These are part of the public API, even if they aren't constructed by users. For example, TS users will want type information when they do:

```ts
light.shadow.mapSize.width = 512;
```

You could make a case for accessing the constructors too, if I want to check `light.shadow instanceof SpotLightShadow` or something, but I don't feel so strongly about that.

Comment (Mugen87) [reply src/Three.d.ts:61]: Okay. Um, but that means TS users don't get type information for `PointLightShadow` right now. How about adding `PointLightShadow` to `Three.d.ts` and just remove the JS exports? Not sure if `light.shadow instanceof SpotLightShadow` is a compelling use case.

Comment (donmccurdy) [reply src/Three.d.ts:61]: Sounds good to me. 👍

Comment (Mugen87) [reply src/Three.d.ts:61]: Updated 👍 .

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**